### PR TITLE
add net.sourceforge.fspclient

### PR DESCRIPTION
**Pull request by hsn10**

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build net.sourceforge.fspclient

File: net.sourceforge.fspclient.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		"buildsystem": "simple",
+		"build-commands": [ "python3 /app/bin/scons prefix=/app mandir=/app/man install" ],
+		"sources": [
+		{
+			"type":"archive",
+			"url":"https://downloads.sourceforge.net/fspclient/fspclient-0.93.1.tar.bz2",
+			"sha256" : "a419870f43eef1d8ab059379e0baddeb6bcc5c8acd26561939fea9be51e75487"
+		}]
+	}],
+	"finish-args": [
```
Comment (barthalion) [inline net.sourceforge.fspclient.json:31]: It's preferred to have it above `modules`, it's easier to review that way.

**Comment by barthalion** [comment]
Your application needs to ship metainfo file. It can be generated with https://bilelmoussaoui.github.io/metainfo-generator/#/ or https://www.freedesktop.org/software/appstream/metainfocreator/#/ and validated locally as described in out documentation: https://github.com/flathub/flathub/wiki/AppData-Guidelines

**Comment by barthalion** [comment]
bot, build net.sourceforge.fspclient

**Comment by barthalion** [comment]
Metainfo file needs some fixes:

```
builddir/files/share/appdata/net.sourceforge.fspclient.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• style-invalid         : Not enough <screenshot> tags, minimum is 1
• tag-missing           : <release> required
• tag-missing           : <description> required
• tag-missing           : <url> is not present
```

**Comment by barthalion** [comment]
bot, build net.sourceforge.fspclient

**Comment by barthalion** [comment]
`<releases>` should look like this:

```
<releases>
    <release date="2020-02-28" version="0.14.1" />
</releases>
```

**Comment by barthalion** [comment]
bot, build net.sourceforge.fspclient

**Comment by barthalion** [comment]
bot, build net.sourceforge.fspclient

**Pull request closed without merging**